### PR TITLE
Update neurodamus family

### PR DESCRIPTION
**Pull request by ferdonline**
With neurodamus-core 2.6.0 comes the following:

- Quit when there is a duplicate target
- BinReportHelper: Ctor dont return value
- Print correct number of target cells and subtargets
- nrnPath can be a file (or a dir: deprecated)
- Attempt finding start.target initially in circuitpath
- Better detection / precedence of connectivity: 1. Sonata, 2. syn2, 3. Nrn
- Improved Deprecation and Error messages

With neurodamusc-core 2.7.0, common mods 1.3, synapsetool 0.5.2 comes

 - population ids to seed synapse RNGs
 - Better support for sonata, with verbose mode

### CODE REVIEW

**Comment by pramodk** [comment]
Please retest

**Comment by pramodk** [comment]
CI is failing because of tags missing in repos.

File: var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class NeurodamusHippocampus(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
 
     version('develop', git=git, branch='master', submodules=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py]:
```suggestion
    version('develop', branch='master', submodules=True, clean=False)
```

File: var/spack/repos/builtin/packages/neurodamus-mousify/package.py [hunk 2]
```diff
@@ -12,5 +12,6 @@ class NeurodamusMousify(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/mousify"
 
     version('develop', git=git, branch='master', submodules=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neurodamus-mousify/package.py]:
```suggestion
    version('develop', branch='master', submodules=True, clean=False)
```

File: var/spack/repos/builtin/packages/neurodamus-neocortex/package.py [hunk 3]
```diff
@@ -12,8 +12,9 @@ class NeurodamusNeocortex(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/neocortex"
 
     version('develop', git=git, branch='master', submodules=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neurodamus-neocortex/package.py]:
```suggestion
    version('develop', branch='master', submodules=True, clean=False)
```

**Review by matz-e — CHANGES_REQUESTED** [review]
Consistency (:

**Comment by ferdonline** [comment]
please retest

**Comment by jorblancoa** [comment]
please retest

**Comment by jorblancoa** [comment]
please retest

File: var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py [hunk 4]
```diff
@@ -12,7 +12,7 @@ class NeurodamusHippocampus(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
 
     version('develop', branch='master', submodules=True, clean=False)
-    version('0.4', tag='0.4', submodules=True, clean=False)
+    version('0.4', tag='0.4-1', submodules=True, clean=False)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py:15]: Already? Not 0.4.1?

Comment (ferdonline) [reply var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py:15]: we cant delete 0.4, so that "notation" goes for "use this one instead"

Comment (matz-e) [reply var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py:15]: Hm, OK. I thought the `-1` would be used for the actual version number for re-deployment. This seems more like a brown-bag/patch release (:

**Review by matz-e — APPROVED** [review]

**Pull request merged**